web/webkit/src/main/scala/net/liftweb/http/HtmlNormalizer.scala

```diff
@@ -0,0 +1,291 @@
+package net.liftweb
+package http
```

Comment:
I'm slowly becoming skeptical of this package organization. We're stuffing a bunch of things in a package named `http` that doesn't deal with the HTTP protocol, which isn't thrilling.

Food for thought.